### PR TITLE
Compute Engine samples

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: computeengine/vms_api.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    callback(null, authClient);
+  });
+}
+// [END auth]
+
+// [START list]
+/**
+ * @param {Function} callback Callback function.
+ */
+function getVmsExample(callback) {
```
Comment (JustinBeckwith) [inline computeengine/vms_api.js]: This looks the same as the example above, just using the api client instead of gcloud.  Why?

**Comment by jonparrott** [comment]
LGTM

File: computeengine/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "description": "Collection of Node.js samples on Google Compute Engine.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "author": "Google Inc.",
+  "engines": {
+    "node": ">=0.10.x"
+  },
+  "scripts": {
+    "sendgrid": "node sendgrid.js",
```
Comment (JustinBeckwith) [inline computeengine/package.json]: nothing in this sample is really compute engine specific - at what point do we just have a single sendgrid sample, and use that for g[cka]e?

Comment (jmdobry) [inline computeengine/package.json]: Yes

Comment (jmdobry) [inline computeengine/package.json]: Just realized that wasn't a yes/no question. 😆

**Pull request merged**